client/tests/simulator_tests_registration/mod.rs

```diff
@@ -0,0 +1,9 @@
+// Please see "simulator_tests.rs" for details.
```

Comment:
Preferably use doc comments and not those styles

```
//! Please see ...
```

Reply:
Fixed